pages/interface/components/PublishedSince/index.js

```diff
@@ -1,16 +1,25 @@
-import { formatDistanceToNowStrict } from 'date-fns';
-import { pt } from 'date-fns/locale';
+import { formatDistanceToNowStrict, format } from 'date-fns';
+import { ptBR } from 'date-fns/locale';
+import { Tooltip } from '@primer/react';
 
-export default function PublishedSince({ date }) {
-  return <span suppressHydrationWarning={true}>{formatPublishedSince(date)}</span>;
+function formatPublishedSince(date) {
+  const publishedSince = formatDistanceToNowStrict(new Date(date), {
+    addSuffix: false,
+    includeSeconds: true,
+    locale: ptBR,
+  });
 
-  function formatPublishedSince(date) {
-    const publishedSince = formatDistanceToNowStrict(new Date(date), {
-      addSuffix: false,
-      includeSeconds: true,
-      locale: pt,
-    });
+  return `${publishedSince} atrás`;
+}
 
-    return `${publishedSince} atrás`;
-  }
+function formatTooltipLabel(date) {
+  return format(new Date(date), "EEEE, d 'de' MMMM 'de' yyyy 'às' HH:mm", { locale: ptBR });
+}
+
+export default function PublishedSince({ date }) {
+  return (
+    <Tooltip sx={{ position: 'absolute', ml: 1 }} aria-label={formatTooltipLabel(date)}>
```

Comment:
```suggestion
    <Tooltip sx={{ position: 'absolute', ml: 1, mt: '1px' }} aria-label={formatTooltipLabel(date)}>
```

Aqui complicou unificar o `Tooltip` no `PublishedSince`, pois sem `mt: '1px'` vai ficar desalinhado no `Content` e com `mt: '1px'` vai ficar desalinhado no `ContentList`.

![image](https://user-images.githubusercontent.com/77860630/222546219-81ba0c95-daf0-4b4a-a771-c574f949d1d1.png)

![image](https://user-images.githubusercontent.com/77860630/222546535-b95fe37b-c439-4fbf-94a6-9a12897b62db.png)

Reply:
Resolvi esse problema adicionando um padding no `Content`.